src/test/java/stepDefinition/HomePageStepDef.java

```diff
@@ -1,24 +1,18 @@
 package stepDefinition;
 
-import cucumber.api.java.en.Given;
-import cucumber.api.java.en.Then;
-import pages.HomePage;
-
-import static helpers.Reflection.getPage;
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
 
+import cucumber.api.java.en.Given;
+
+import static helpers.ElementSearchUtils.getPage;
+
 public class HomePageStepDef extends AbstractStepDef {
 
     @Given("^user is on Home page$")
     public void userIsOnHomePage() {
         webDriver.navigate().to(fileReader.getApplicationUrl());
-        assertThat("Page is displayed", webDriver.getCurrentUrl(), is(getPage(HomePage.class.getSimpleName()).getPageUrl()));
+        assertThat("Page is displayed", webDriver.getCurrentUrl(), is(getPage("HomePage").getPageUrl()));
```

Comment:
you can use getPageObjectName method instead of string, cause if it will be a class you can easy perform refactor

Reply:
is this necessary for for a step definition which purpose is only to serve as a precondition for "user is on Home page" ?